nights/2015-11-01-README.md

```diff
@@ -1,32 +1,24 @@
 # 2015-11-01
 
-## Reviewed by:   your_name_here
+## Reviewed by:   Isobel Snellenberger
 
 ## Unusual images?
 
 Are there any images that look unusual? List the file name of any unusual images for this night here, with description:
 
-+ `crazy-object-001R.fit` -- not sure what this is an image of
++ ` calibration-004D60.fit ` -- a few really bright spots in bottom left
++ ` calibration-007D10.fit ` -- a few really bright spots in top of image
++ ` calibration-008D10.fit ` -- bright spots in the lower left
 + `another-crazy-object-003B.fit` -- this looks more like a flat than a science image.
 + `full-moon-005B.fit` -- nice satellite or plane track in here.
 
-Delete this list if there are no unusual images.
-
 ## Missing information?
 
 Check these off if they are true:
 
 - [ ] No images are missing filter information (except BIAS and DARK, which need no filter).
 - [ ] No images are missing pointing information (RA/Dec and WCS)
 - [ ] No images are missing object names (only applies to science images)
```

Comment:
Please fill in these checkboxes: `[ ]` by putting an `x` in them like this: `[x]`